packages/react-core/src/components/Wizard/Wizard.tsx

```diff
@@ -123,18 +111,9 @@ export class Wizard extends React.Component<WizardProps, WizardState> {
     appendTo: null as HTMLElement
   };
   private container: HTMLDivElement;
-  private titleId: string;
-  private descriptionId: string;
-  private isModal: boolean;
 
   constructor(props: WizardProps) {
     super(props);
-    const newId = Wizard.currentId++;
-    this.isModal = !props.isInPage;
-    if (this.isModal) {
-      this.titleId = `pf-wizard-title-${newId}`;
-      this.descriptionId = `pf-wizard-description-${newId}`;
-    }
 
     this.state = {
```

Comment:
If all you're doing is setting state, you don't need a constructor and can just set the class member `state`.